src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -173,6 +186,9 @@ foam.CLASS({
               await this.nextStep();
             }
           } catch (err) {
+              if ( this.DuplicateEmailException.isInstance(err.data.exception) ) {
```

Comment:
so you added the exception you wanted for this path... but this error is also currently thrown in foam/nanos/auth/RetrievePassword.js
foam/nanos/auth/ruler/PreventDuplicateEmailAction.js
nanopay/auth/NanopayResetPasswordTokenService.js

can you guarantee the correct flow by just checking the type/instance ? Perhaps also checking the message...